src/js/view/controls/controlbar.js

```diff
@@ -216,6 +217,7 @@ export default class Controlbar {
         }, () => {
             feedShownId = '';
         });
+        setAttribute(nextElement, 'dir', 'auto');
```

Comment:
What does this do?

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/dir
"The dir global attribute is an enumerated attribute indicates the directionality of the element's text"
"dir=auto lets the user agent decide. It uses a basic algorithm as it parses the characters inside the element until it finds a character with a strong directionality, then applies that directionality to the whole element."